easybuild/easyconfigs/l/LightGBM/LightGBM-4.5.0-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = "LightGBM"
+version = "4.5.0"
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = "https://lightgbm.readthedocs.io"
+description = """A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM
+or MART) framework based on decision tree algorithms, used for ranking,
+classification and many other machine learning tasks."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('scikit-build-core', '0.9.3'),
+    ('wget', '1.24.5'),
+]
+
+dependencies = [
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Arrow', '14.0.1'),  # optional
+    ('dask', '2023.9.2'),  # optional
+    ('scikit-learn', '1.3.1'),  # optional
+]
+
+use_pip = True
+
+# example files are not distributed with the sources
+_test_repo_url = "https://raw.githubusercontent.com/microsoft/LightGBM/refs/heads/master/examples"
```

Comment:
let's use a version of the examples too instead of master

Reply:
good point, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/21699/commits/d6a2e48763ba3577cc0e89112a429094bd28e8fa